earthfile2llb/interpreter.go

```diff
@@ -787,6 +787,10 @@ func (i *Interpreter) handleSaveArtifact(ctx context.Context, cmd spec.Command)
 	saveTo = i.expandArgs(saveTo, false)
 	saveAsLocalTo = i.expandArgs(saveAsLocalTo, false)
 
+	if saveAsLocalTo == "." {
+		saveAsLocalTo = "./"
+	}
```

Comment:
Mind moving this to the converter? Best to keep the interpreter as agnostic as possible.